recipes/package.rb

```diff
@@ -19,16 +19,10 @@
 # limitations under the License.
 #
 
-if platform_family?('rhel', 'debian', 'amazon', 'suse')
-  package node['php']['packages'] do
+node['php']['packages'].each do |pkg|
+  package pkg do
```

Comment:
We can probably just pass the whole array in here now that all our platforms support multipackage installs